.github/workflows/bazelBuildAndTest.yml

```diff
@@ -0,0 +1,30 @@
+name: Bazel Build and Test
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
who does this notify when it fails?

Reply:
My understanding this is for running this workflow on new commits on main branch.
Setting notifications and if this workflow is merge blocking or not isn't specified here If I understand GH actions documentations correctly.